docs/cip/README.md

```diff
@@ -1775,57 +1717,82 @@ for further discussion.
-| Mean committee size                           |         $n$         |   600 stakepools   | Modeling of the proposed certificate scheme indicates that certificates reach their minimum size of ~8 kB at this committee size, given a realistic distribution of stake among pools.      |
-| Quorum size                                   |       $\tau$        |        60%         | Quorum should be enough greater than 50% that sortition fluctuations do not give a 50% adversary a temporary majority, but low enough that weak adversaries cannot thwart an honest quorum. |
-
-<em>Table 6: Feasible Protocol Parameters</em>
+| Parameter                                     |       Symbol        |   Feasible value   | Justification                                                                                                                                                                          |
+| --------------------------------------------- | :-----------------: | :----------------: | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| RB header diffusion bound                     | $\Delta_\text{hdr}$ |       1 slot       | Must be faster than full RB diffusion for equivocation detection; simulations show headers reach all nodes within 1 slot due to smaller size.                                          |
+| Voting period length                          |   $L_\text{vote}$   |      7 slots       | Short stages increase settlement speed, but the stage length must be generously larger than the propagation time for EBs.                                                              |
+| Diffusion period length                       |   $L_\text{diff}$   |     5-10 slots     | Calculated as $\Delta_\text{EB}^{\text{wc}} - \Delta_\text{RB} - L_\text{vote}$. With typical values: ~20 - 5 - 7 = 8 slots                                                            |
+| Endorser-block referenceable transaction size |  $S_\text{EB-tx}$   |       12 MB        | Simulations indicate that 200 kB/s throughput is feasible at this block size.                                                                                                          |
+| Endorser block max size                       |    $S_\text{EB}$    |       512 kB       | Endorser blocks must not be so large that they cannot diffuse and be validated within the voting period $L_\text{vote}$.                                                               |
+| Maximum Plutus steps per endorser block       |          -          |  2000G step units  | Simulations at high transaction-validation CPU usage.                                                                                                                                  |
```

Comment:
We'll need to lower this to 1000 Gstep or 600 Gstep, based on the simulation using `sim-rs-1.2.0`. I'll fix this in a separate PR.